language/return_spec.rb

```diff
@@ -343,100 +360,99 @@ def f
 
             puts "after begin"
           END_OF_CODE
+        end
 
-          $?.exitstatus.should == 0
+        it "does not fire ensure block before returning while loads file" do
```

Comment:
@eregon it looks like a Ruby bug and I am not sure this case should be present. Previous not changed case illustrate the feature

Reply:
That looks like a bug indeed. I reported it at https://bugs.ruby-lang.org/issues/14061.
Please annotate it with `ruby_bug "#14061", "2.4"` and specify the opposite behavior inside, i.e., the expected non-buggy behavior.